src/components/Tooltip/index.tsx

```diff
@@ -7,22 +7,25 @@ type TooltipPosition = 'top' | 'bottom' | 'right' | 'left';
 interface Props {
   content: React.ReactNode | string;
   children: React.ReactElement;
+  disabled?: boolean;
   position: TooltipPosition;
   variablesClassName?: string;
 }
 
 const Tooltip: React.FC<Props> = props => {
-  const { content, children, position, variablesClassName } = props;
+  const { content, children, disabled = false, position, variablesClassName } = props;
 
```

Comment:
There's no need to set the disabled prop to false. Could you please remove this default value?